### ISSUE TITLE
Ability to tunnel an HTTP service over Mumble 

### ISSUE THREAD

**Issue by EternityForest** [mumble-voip/mumble#6249]
### Context

Mumble is one of the few self hosted projects that is actually easy to use, due to it's lack of reliance on traditional PKI.

As so much of the functionality is already there,  it doesn't seem to hard to add the ability to share a locally hosted website via the Mumble protocol.

This would allow for a huge number of additional features, the most relevant of which might be admin pages and self registration.

### Description

This would be a VPN-like tunneling service, that would allow a website hosted on the Mumble server machine to appear as if it were on a subdomain of the client's localhost.

The server admin would configure one or more web services, and set up permissions. They could make them public, or only accessible to logged-in members.

The client app would show these services, allow the user to select one, and launch the browser.

The Mumble client and server would then act as a tunneling proxy.

This would allow Mumble to support Forums, personal sites, social networks, video conferencing, IoT device access, and many other features that are sometimes found in chat-like contexts.

It would also make it a natural choice for FOSS development, as it could be used for testing.

As subdomains would be used, the browser would handle any security and sandboxing issues.

One possibly way to implement it would be to allow clients to share sites, assuming the server settings allowed it.   Imagine a game development community with forums, and user created games themselves all done through Mumble!

### Mumble component

Both

### OS-specific?

No

**Comment by Krzmbrzl** [comment]
No. Adding webserver capabilities to Mumble will open up a whole new can of worms of required features and most of all: security considerations.

Mumble is a VoIP application and it should focus on exactly that. Delivering web pages should be done with a dedicated webserver. There are good options available with Apache or Nginx.
If you want to tie your webpage expierence with Mumble, use the Ice RPC interface to e.g. query registered users.
